### DISCUSSION TITLE
idea 2021.2.2 an error occurred while fetching the completions

### DISCUSSION THREAD

**Discussion by affirmatives** [community/community#18158] [Copilot Conversations]
<img width="495" alt="WX20220610-002941@2x" src="https://user-images.githubusercontent.com/64325902/172898086-586897b5-5ca7-4d6f-8e0e-e0350a5d451d.png">

its not work ..

**Comment by bmuskalla** [comment]
What does copilot say when you click the icon in the status bar at the bottom? 
Do you have the most recent plugin version installed?

**Reply by yxqweasd** [reply]
have a look at the right bottom icon(put your mouse on it, it will show you the status of copilot) 
maybe try upgrade the copilot extension version.
